### PR TITLE
Fix: #454

### PR DESCRIPTION
**Pull request by dongweiming**
## What do these changes do?

Compatible with Python3.8's IncompleteReadError

## Related issue number

https://github.com/aio-libs/aiomysql/issues/454

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

**Comment by Dazix** [comment]
@asvetlov can you look to this? We have python3.8 app which fails because of this issue.

**Comment by remy-sl** [comment]
Guys, we are stuck on this as well. All I can do is post this humble bump with a kind request to expedite a release with this fix. Thanks in advance.

**Comment by matino** [comment]
@asvetlov Can we have it merged and get a new version released? Python 3.8 is out for some time now, it's a real shame that aiomysql is lagging far behind...

**Comment by NotSoSuper** [comment]
How is this not fixed yet...

**Comment by Bu3a3a** [comment]
Hey guys! Any update on the issue? Please merge it asap.

**Comment by Dazix** [comment]
Can someone else look at this please? @webknjaz @terrycain @jettify

File: aiomysql/connection.py [hunk 1]
```diff
@@ -596,7 +600,7 @@ def write_packet(self, payload):
     async def _read_bytes(self, num_bytes):
         try:
             data = await self._reader.readexactly(num_bytes)
-        except asyncio.streams.IncompleteReadError as e:
+        except IncompleteReadError as e:
```
Comment (webknjaz) [inline aiomysql/connection.py:603]: From the docs, it looks like it should be
```suggestion
        except asyncio.LimitOverrunError as e:
```
and no imports needed.

Comment (Dazix) [reply aiomysql/connection.py:603]: @dongweiming pls

Comment (dongweiming) [reply aiomysql/connection.py]: @webknjaz @Dazix fixed  👍

**Comment by webknjaz** [comment]
Merged, somebody needs to fix the CI tho

**Comment by Dazix** [comment]
@webknjaz one last thing, could you publish new version with these changes to pypi.org? Thanks.

**Comment by webknjaz** [comment]
Only when someone fixes the CI

**Pull request merged**